erizo/src/erizo/rtp/BandwidthEstimationHandler.cpp

```diff
@@ -88,6 +88,9 @@ void BandwidthEstimationHandler::notifyUpdate() {
 }
 
 void BandwidthEstimationHandler::process() {
+  if (!initialized_) {
```

Comment:
we're calling process() from read() which is safe now, so I think we don't need this line.